.ko.yaml

```diff
@@ -1,7 +1,2 @@
-# Use the Jenkins base image as it is a public image that
-# reliably sets up and uses a non-root user (uid 1000).
-# https://github.com/jenkinsci/docker/blob/master/Dockerfile-slim#L15
-# Ideally we should use an unprivileged distroless user if
-# https://github.com/GoogleContainerTools/distroless/issues/235 is implemented.
-baseImageOverrides:
-        github.com/knative/serving/test/test_images/runtime-unprivileged: jenkins/jenkins:lts-slim
```

Comment:
I think we still need this for our conformance testing.

Reply:
cc @dgerd we could probably change it now though.

Reply:
Looking through the code we actually should be able to use a non-root user for all tests just fine.

[TestShouldRunAsUserContainerDefault](https://github.com/knative/serving/blob/master/test/conformance/runtime/user_test.go#L73) needs to be updated though.

You should be able to change line 76 in that file to just use 'fetchRuntimeInfo", and update line 32 to the uid of the nonroot user to get the test working again.

Since all tests will be using an unprivileged user you should be able to delete [fetchRuntimeInfoUnprivileged](https://github.com/knative/serving/blob/master/test/conformance/runtime/util.go#L34), and the constant [here](https://github.com/knative/serving/blob/84f8bb8321ce0f71391eba5ff76b1e95c12eb68c/test/conformance.go)

Reply:
Thanks for the pointers @dgerd. With my latest commit all e2e and conformance tests passed for me locally.